core/Concurrency/Lock.php

```diff
@@ -21,12 +21,19 @@ class Lock
 
     private $lockKey   = null;
     private $lockValue = null;
+    private $defaultTtl = null;
 
-    public function __construct(LockBackend $backend, $lockKeyStart)
+    public function __construct(LockBackend $backend, $lockKeyStart, $defaultTtl = null)
     {
         $this->backend = $backend;
         $this->lockKeyStart = $lockKeyStart;
         $this->lockKey = $this->lockKeyStart;
+        $this->defaultTtl = $defaultTtl;
+    }
+
+    public function reexpireLock()
```

Comment:
should it be maybe `reExpireLock`? Not too important though...

Reply:
Yeah, it looks weird to me too. But so did `reExpireLock` and I couldn't choose :)